racket/collects/file/private/glob.rkt

```diff
@@ -388,6 +399,47 @@
             (string c)])))
       (string-join str* ""))))
 
+(define GLOB-WILDCARD-CHAR* '(#\* #\? #\[ #\] #\{ #\} #\,))
```

Comment:
Since backslashes have special meaning as "escape the next char", backslashes need to be escaped too, turning `(string #\\)` into `(string #\\ #\\)`, right?

Reply:
I think this was wrong, `#\\` has no meaning by itself but things like `(string #\\ #\*)` have special meaning.